docs/getting-started.md

```diff
@@ -13,6 +13,15 @@ import TabItem from '@theme/TabItem';
 
 # Getting Started with Momento Cache and Momento Topics
 
+TESTING SOME DYNAMICALLY INJECTED EXAMPLE CODE:
+
+```javascript
+%%%example:typescript:code:ListCaches%%%
+```
+
+%%%example:go:description:Logging%%% %%%example:go:code:Logging%%%
+
+
```

Comment:
This is what I'm currently imagining for how we would specify the example snippets that we wanted to inject.